interface/modules/custom_modules/oe-module-visual-ehr/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
No need for this link, instead point your DOCBLOCK @license value to the main GPL link

``` php
/**
 * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
 */
```